src/libraries/Common/tests/System/Net/Http/Http2LoopbackConnection.cs

```diff
@@ -631,6 +631,23 @@ public async Task PingPong()
             Assert.Equal(pingData, pingAck.Data);
         }
 
+        public async Task<PingFrame> ReadPingAsync(TimeSpan timeout)
+        {
+            Frame frame = await ReadFrameAsync(timeout).ConfigureAwait(false);
+            Assert.NotNull(frame);
+            Assert.Equal(FrameType.Ping, frame.Type);
+            Assert.Equal(0, frame.StreamId);
+            Assert.False(frame.AckFlag);
```

Comment:
We should also assert a payload size of 8.

Reply:
Done.